src/averageAges.js

```diff
@@ -20,6 +20,22 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  // const MEN = people.filter(person => century
+  //   ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+  //   : person.sex === 'm'
+  // );
+
+  const MEN = century
```

Comment:
```suggestion
  const men = century
```
Wrong naming. UPPERCASE_WITH_UNDERSCORES should only be used for literal constants that will never change, here it's just a variable that will never be reassigned. Such variables should be named in lowerCamelCase style.